public/viewControler/viewControler.js

```diff
@@ -0,0 +1,54 @@
+import { components } from "../view/index.js";
+
+const changeView = (route) => {
+  const root = document.getElementById("root");
+  const footer = document.getElementById("footer-container");
+  const header = document.getElementById("header-container");
+  root.innerHTML = "";
+  footer.innerHTML = "";
+  header.innerHTML = "";
+  switch (route) {
+    case "#/": {
+      header.appendChild(components.Header());
+      return root.appendChild(components.Home());
+    }
```

Comment:
Al final de cada caso lleva break;   les recomiendo revisar la sentencia de control switch.
```suggestion
    break;
    }
```

Reply:
y así en todas